static/app/components/replays/replayCurrentUrl.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import TextCopyInput from 'sentry/components/forms/textCopyInput';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+import space from 'sentry/styles/space';
+import getCurrentUrl from 'sentry/utils/replays/getCurrentUrl';
+
+function ReplayCurrentUrl() {
+  const {currentTime, replay} = useReplayContext();
+  if (!replay) {
+    return null;
+  }
+
+  return (
+    <UrlCopyInput>
+      <TextCopyInput>{getCurrentUrl(replay, currentTime)}</TextCopyInput>
+    </UrlCopyInput>
+  );
+}
+
+const UrlCopyInput = styled('div')`
```

Comment:
Ideally we'd style `TextCopyInput` instead of adding a wrapping div here, but we'd have to refactor `TextCopyInput` to pass `className` to `<Wrapper>` in there.

Reply:
Sorry I see this now, I moved too fast and didn't see the comment.

Reply:
done! that's so cool how we can style the internals too TIL 👍

Reply:
the updated html looks the same visually, but the classnames are merged into on div:

<img width="705" alt="Screen Shot 2022-05-06 at 10 36 55 PM" src="https://user-images.githubusercontent.com/187460/167154975-274e6a39-9a0a-4140-98ae-e613f2110099.png">